js/adapt-contrib-gmcq.js

```diff
@@ -42,7 +25,7 @@ define([
             }, this));
```

Comment:
don't suppose you'd mind changing this to `this.setReadyStatus.bind(this)` while you're here?

Reply:
No worries 👍